docs/Commands.md

```diff
@@ -0,0 +1,21 @@
+# Commands
+
+[vscode-xml](https://github.com/redhat-developer/vscode-xml) provides several vscode commands which are available with `Ctrl+Shift+P`.
+
+![XML Commands](images/Commands/XMLCommands.png)
+
+## Open XML Documentation
+
+This command opens the `XML Documentation`
+
+## Validate current XML file
+
+This command re-triggers the [XML Validation](Validation.md#xml-validation) for the current file.
+
+When the [Server Cache Path](Preferences.md#server-cache-path) is activated, the command removes the referenced XSD, DTD grammar from the local cache.

```

Comment:
@datho7561 do you think the description is understandable?

Reply:
This description makes sense. The description in the package.json is the one that I think needs improving.